detox/test/src/Screens/index.js

```diff
@@ -51,5 +52,6 @@ export {
   DeviceScreen,
   ElementScreenshotScreen,
   WebViewScreen,
-  VirtualizedListStressScreen,  
+  VirtualizedListStressScreen,
```

Comment:
Isn't this out of context?

Reply:
I just added it at the last position because it's new. I don't necessarily think it should be the same position it shows up it the actual main screen with all the titles. Doesn't really matter to me, I can change it if you want